README.md

```diff
@@ -87,7 +87,7 @@ If you want to run FormPlayer in Docker as a service of CommCare HQ, follow thes
 2. Start Docker for CommCare HQ as usual, either with `scripts/docker up` (services only) or `scripts/docker runserver` (HQ and services).
 3. Run `scripts/get_webhost` and append its output to `/etc/hosts`. (Or if you run Linux, `scripts/get_webhost | sudo tee -a /etc/hosts`.)
 4. If you run CommCare HQ in Django locally:
-   1. Update localsettings.py with `FORMPLAYER_URL = 'http://localhost:8010'`
+   1. Update localsettings.py with `FORMPLAYER_URL = 'http://localhost:8010'` and `BASE_ADDRESS = 'http://webhost:8000'`
```

Comment:
I think you need to leave out the `http://` part here

Reply:
whoops you're completely right thanks

Reply:
(fixed)